src/sail_sv_backend/jib_sv.ml

```diff
@@ -2386,8 +2412,12 @@ module Make (Config : CONFIG) = struct
     let block_terminator last = if last then semi else semi ^^ hardline in
     let pp_body = function
       | SVS_aux (SVS_block statements, _) ->
-          concat (Util.map_last (fun last -> pp_statement ~terminator:(block_terminator last)) statements)
-      | statement -> pp_statement ~terminator:semi statement
+          concat
+            (Util.map_last
+               (fun last -> pp_statement ?prepend_globals:None ~terminator:(block_terminator last))
+               statements
+            )
+      | statement -> pp_statement ?prepend_globals:None ~terminator:semi statement
```

Comment:
There's no need to do `?optional_argument:None` in OCaml (just omit it). I'm actually very surprised this even compiles, as I thought an optional argument was only implicitly an option type if you don't give a default value.